lib/client.js

```diff
@@ -79,6 +82,11 @@ module.exports = function (dependencies) {
               resolve(this.write(notification, device, retryCount + 1));
               return;
             }
+            else if (status === "500" && response.reason === "InternalServerError") {
```

Comment:
Can you make formatting consistent with the rest of the file and make it

`} else if (...) {`